src/VisualStudio/Core/SolutionExplorerShim/DiagnosticItem/DiagnosticItem.BrowseObject.cs

```diff
@@ -20,7 +20,7 @@ public BrowseObject(DiagnosticItem diagnosticItem)
                 _diagnosticItem = diagnosticItem;
             }
 
-            [DisplayName("ID")]
+            [BrowseObjectDisplayName("DiagnosticItemIDDisplayName")]
```

Comment:
Can we replace the string arguments here with strongly typed arguments? For example, `nameof(SolutionExplorerShim.DiagnosticItemIDDisplayName)`